src/global_styling/variables/_form.scss

```
@@ -36,6 +36,7 @@ $euiFormCustomControlDisabledIconColor: shadeOrTint($euiColorMediumShade, 38%, 4
 $euiFormCustomControlBorderColor: shadeOrTint($euiColorLightestShade, 18%, 30%) !default;
 $euiFormControlDisabledColor: $euiColorMediumShade !default;
 $euiFormControlBoxShadow: 0 1px 1px -1px transparentize($euiShadowColor, .8), 0 3px 2px -2px transparentize($euiShadowColor, .8) !default;
+$euiFormControlPlaceholderText: $euiColorDarkShade !default;
```

I just had a thought, we should really be using a text color variant and contrast function for this so ensure it maintains proper contrast if the globals are adjusted. Since the background color of inputs are darker than white, we'll want to use that as the background parameter. And I think we can start with the subdued text color since we want it to be lighter than regular text.

```suggestion
$euiFormControlPlaceholderText: makeHighContrastColor($euiTextSubduedColor, $euiFormBackgroundColor) !default;
```